v3/v3_service.go

```diff
@@ -1777,9 +1752,9 @@ func (op Operations) ListAllUserGroup(ctx context.Context, filter string) (*User
 		for hasNext(&remaining) {
 			resp, err = op.ListUserGroup(ctx, &DSMetadata{
 				Filter: &filter,
-				Kind:   utils.StringPtr("user"),
-				Length: utils.Int64Ptr(itemsPerPage),
-				Offset: utils.Int64Ptr(offset),
+				Kind:   ptr.To("user_group"),
```

Comment:
Kind was 'user' now it's 'user_group'. Was it a bug? (If it's bug please add info to PR description)
Just want to be sure that it's not typo or mistake here.